testresources/__init__.py

```diff
@@ -187,6 +194,8 @@ def _strongly_connected_components(graph, no_resources):
 class OptimisingTestSuite(unittest.TestSuite):
     """A resource creation optimising TestSuite."""
 
+    known_suite_classes = None
```

Comment:
I realise this can never happen, but it seems odd to me to set this to None, only to treat it as an iterable a few lines further down. I'd be happier if this were set to an empty tuple.... Yes, I realise we're always setting it to be a tuple later on, but still....

Consider this a mild suggestion - feel free to ignore if you disagree.